src/GraphODataTemplateWriter/CodeHelpers/TypeScript/TypeHelperTypeScript.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+
+namespace Microsoft.Graph.ODataTemplateWriter.CodeHelpers.TypeScript
+{
+    using Vipr.Core.CodeModel;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public static class TypeHelperTypeScript
+    {
+
+        // enum value string, ex: "low" | "normal" | "high"
+        public static String GetEnumValues(this OdcmEnum _enum) {
+            String enumVals = null;
+            foreach (var member in _enum.Members)
+            {
+                if (enumVals == null)
+                {
+                    enumVals = '"' + member.Name + '"';
+                } else
+                {
+                    enumVals += " | " + '"' + member.Name + '"';
+                }
+            }
```

Comment:
Sorry. Aggregate not join, and the syntax is slightly different. I was crossing my .NET with some JavaScript.